app/src/main/java/com/example/playlistmaker/media/data/PlaylistsRepositoryImpl.kt

```diff
@@ -32,21 +32,38 @@ class PlaylistsRepositoryImpl(
         emit(convertFromPlaylistEntity(playlists.reversed()))
     }
 
-    override fun putPlaylist(playlist: Playlist) {
+    override fun putPlaylist(playlist: Playlist, tracks: List<Track>) {
+        //Восстанавливаем таблицу треков
+        Log.d("getting tracks", "$tracks")
+        for (track in tracks) {
+            if (track.trackId != 0.toLong()) {
```

Comment:
```suggestion
            if (track.trackId != 0L) {
```
Можно вот так писать